src/components/badges/_badges.scss

```diff
@@ -15,7 +15,7 @@ $includeHtml: false !default;
     @include fixText($badgeFontSize);
     @include uppercaseText;
     overflow: visible; // allow badges align as text
-    min-height: auto;
```

Comment:
`An earlier revision of the spec added the auto keyword and used it as the initial value. The CSSWG subsequently resolved to revert this change, however. As of March 29, 2013, the latest Editor's Draft doesn't modify the min-width property anymore (i.e. it no longer introduces the auto value).` so it will not get any support